extensions/arc/src/ui/dashboards/postgres/postgresExtensionsPage.ts

```diff
@@ -60,8 +61,8 @@ export class PostgresExtensionsPage extends DashboardPage {
 		}).component();
 
 		const link = this.modelView.modelBuilder.hyperlink().withProperties<azdata.HyperlinkComponentProperties>({
-			label: loc.extensionsLearnMore,
```

Comment:
Remove this string if it's no longer used

Reply:
Being used in extensions/arc/src/ui/dialogs/addPGExtensionsDialog.ts